it/test/helpers/WiremockHelper.scala

```diff
@@ -148,6 +148,17 @@ object WiremockHelper extends Eventually with IntegrationPatience {
       )
     )
 
+  // for now overload the stubPut because there are quite a lot of other tests which do not have the request body supplied
```

Comment:
do you want me to clean up the other tests or create a newly named stub for this or leave as is? can remove the comments.

Reply:
We could do this as part of other cleanup ticket as this is not in scope of this ticket.

Reply:
awesome can leave for now